### PR TITLE
add highlight to the right row when adding/deleting new rows

### PR DESCRIPTION
**Pull request by Levilian**
Fix: 
#1525 
#1558

### CODE REVIEW

**Comment by Levilian** [comment]
Adding rows: https://recordit.co/kB1B7LYeZE

**Comment by Levilian** [comment]
Deleting rows: https://recordit.co/08cKS9MJzv

**Comment by Levilian** [comment]
Edges cases: 
deleting the last row
adding the first row
https://recordit.co/6DuT6v0esE

File: frontend/src/components/worksheets/Worksheet.js [hunk 1]
```diff
@@ -718,8 +718,10 @@ class Worksheet extends React.Component {
                         // If the number of bundles decreases, then focus should be on the same bundle as before
                         // unless that bundle doesn't exist anymore, in which case we select the closest bundle that does exist,
                         // where closest means 'next' by default or 'last' if there is no next bundle.
-                        var focus = this.getFocusAfterBundleRemoved();
+                        var focus = this.getFocusAfterBundleRemoved(items);
                         this.setFocus(focus[0], focus[1]);
+                    } else {
+                        this.setFocus(this.state.focusIndex + 1, this.state.subFocusIndex);
```
Comment (yipenghe) [inline frontend/src/components/worksheets/Worksheet.js]: Overall looks good, can you add a comment here explaining when this else condition happens and why the setFocus indices? (like the comments from 718-720) The code is getting more complex so would be good to have comments to follow

**Review by yipenghe — CHANGES_REQUESTED** [review]
Thanks for recording the examples! Just add the comments and good to go

**Review by yipenghe — APPROVED** [review]
Add the comments before merging then good to go

File: frontend/src/components/worksheets/Worksheet.js (hunk 1 above)
Comment (Levilian) [reply frontend/src/components/worksheets/Worksheet.js]: Thanks Yipeng!

**Pull request merged**